recipes/ncvue/ncvue

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
```

Comment:
This file should go in your project as `src/ncvue/__main__.py`. This allows anyone to launch your app by calling `python -m nvcue`.